.github/ISSUE_TEMPLATE/feature-request.yml

```
@@ -0,0 +1,43 @@
+name: Feature Request
+description: Suggest a new feature or an idea for ACCESS-Hive Docs.
+labels: ['new feature']
```

Do you think it could be useful to also set a "FEATURE" prefix to the issue title, similarly to the "BUG" issues template? 
```suggestion
labels: ['new feature']
title: '[FEATURE]: '
```

---

Yeah.. good idea! Initially added it, but then removed it - since I was not sure if it adds anything meaningful to the issue title. Each and every issue would start with `FEATURE:`, that might make the title longer. It's helpful to have only the concise title for issues so it is easy to read when skimming through the project board. Very happy to make this change if we agree that it really adds value to the project board. 

The `BUG:` title makes more sense, as it is just 3 characters and important to flag which issues are bugs on the project board.

---

Yes I understand you point. In general, to identify which issues are bugg, features, etc. we can use labels (that's why we're creating these issue templates in the first place). 
So I agree it's not necessarily needed. 
Since it makes the title longer I agree it might be better not to include it in this case.